openlibrary/templates/work_search.html

```diff
@@ -129,30 +129,30 @@
       <button type="submit" class="larger" id="searchsubmit">$_("Search")</button>
       <span class="mode-options">
         <input type="radio" name="mode" value="everything" id="mode-search-everything" class="search-mode">
-        <label for="mode-search-everything">Everything</label>
+        <label for="mode-search-everything">$_('Everything')</label>
         <input type="radio" name="mode" value="ebooks" id="mode-search-ebooks" class="search-mode" $('checked="checked"' if 'has_fulltext' in param else '')>
-        <label for="mode-search-ebooks">Ebooks</label>
+        <label for="mode-search-ebooks">$_('Ebooks')</label>
         <input type="radio" name="mode" value="printdisabled" id="mode-search-printdisabled" class="search-mode">
-        <label for="mode-search-printdisabled">Print Disabled</label>
+        <label for="mode-search-printdisabled">$_('Print Disabled')</label>
       </span>
         $ sticky = set(['author_facet', 'language', 'first_publish_year', 'publisher_facet',  'subject_facet', 'person_facet', 'place_facet', 'time_facet', 'public_scan_b'])
 
         $for k, values in param.items():
             $if k not in sticky:
                 $continue
             $for v in values if isinstance(values, list) else [values]:
-                <input type="hidden" name="$k" value="$v.replace('"', '&quot;')" />
+                <input type="hidden" name="$k" value="$v.replace('&quot;', '&quot;')" />
```

Comment:
Isn't this a no-op now?

Reply:
Yup, my bad. emacs gets confused by the nested quotes and I messed up fixing it.